src/@types/global.d.ts

```diff
@@ -185,4 +186,19 @@ declare global {
             parameterDescriptors?: AudioParamDescriptor[];
         }
     );
+
+    // eslint-disable-next-line no-var
+    var grecaptcha:
+        | undefined
+        | {
+              reset: (id: string) => void;
+              render: (
+                  divId: string,
+                  options: {
+                      sitekey: string;
+                      callback: () => void;
+                  }
+              ) => string;
+              isReady: () => boolean;
+          };
```

Comment:
Can we please use the DefinitelyTyped lib for this instead of maintaining our own, we avoid this wherever possible.

https://www.npmjs.com/package/@types/grecaptcha

Reply:
@t3chguy those types are for https://www.npmjs.com/package/grecaptcha which isn't the official way of doing it and they are only package-types, not global type definition.

Reply:
Hmm, that's annoying if that's the case given the summary of 

> This package contains type definitions for Google Recaptcha (https://www.google.com/recaptcha).

But looking at the actual types you are right